src/test/kotlin/com/_404/cosmos/springpicnic/ScenarioTests.kt

```diff
@@ -0,0 +1,71 @@
+package com._404.cosmos.springpicnic
+
+import com._404.cosmos.springpicnic.scenario.entity.Scenario
+import com._404.cosmos.springpicnic.scenario.repository.ScenarioRepository
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+
+@Suppress("No" +
+        "nAsciiCharacters", "TestFunctionName")
```

Comment:
나는 H2Tests 작성할 때 인텔리제이에서 노란줄이 뜨길래 추가했던건데.. 이걸 매번 추가하는게 나을지 조금 고민중..
일단 엔터는 없어도 될거같음~
```suggestion
@Suppress("NonAsciiCharacters", "TestFunctionName")
```

Reply:
복붙하는 과정에서 엔터가 들어갔나 보네요 수정하겠습니다.